### DISCUSSION TITLE
Transpose vs tune

### DISCUSSION THREAD

**Discussion by afstudios** [sfztools/sfizz#1159] [Requests]
At the moment it is not possible to use the transpose correctly, as it works as tune actually.
It would be great if the transpose or note shift from V2 could be integrated so the notes could be shifted and tuned in the same time 
Also octave transpose / note shift from V2 does not work.
The opcodes that does not work:

note_offset
octave_offset

Any chance of implementing this?

**Comment by redtide** [comment]
~~This should be moved to issues:~~ is this still a problem?

**Comment by redtide** [comment]
Moved to #1220
